### ISSUE TITLE
Please Transform This Script into a Simple GUI Program

### ISSUE THREAD

**Issue by JoeChungusThrows** [obskyr/khinsider#92]
I'm very happy a script has been developed, but a lot of us are struggling here. It took me 15 minutes to get this working, even though I've gotten it to work in the past. Something like this could be very trivialized if it was a nice, simple GUI.

I would assume a GUI wouldn't take much effort and time to learn and implement.
Even for experienced developers, accessibility matters in any program.
Less people would be complaining.
If anything, I might just fork this project for myself and make a GUI.
But regardless, please just do it.

**Comment by RazzeeX** [comment]
Yes, it is not even remotely intuitive.

**Comment by ExperimentalSubject-8C** [comment]
You just have to read `readme.md`, it's not that difficult to understand unless you are illiterate. The whole point of this project is to remain simple, not to have extra, unnecessary code. the most you have to understand is to use `cd` in command prompt. Then you can just copy/paste `khinsider.py (_insert project url here_)`

**Comment by ks202005** [comment]
No, you definitely need to know more than ``cd``, because you need to install Python and other dependencies through ``pip``. The last time I had to install this fresh I had difficulty installing Beautiful Soup; I don't remember exactly what the issue was, but I'm not a novice and it was throwing me for a loop.

I think if a GUI version is out of the question, an executable CLI version would suffice for novice users. I don't see that being particularly difficult to make (youtube-dl and its forks like yt-dlp are far more complex and do that just fine), and I really don't see why so many projects using Python presume everyone is going to have Python installed or know how to do that to begin with.

**Comment by adyos7498** [comment]
I just have a helper python program that is literally this:
`import khinsider`
`khinsider.download("your-soundtrack-here-last-part-of-url")` 
so i can just open in VSCode and run it, no need to CMD :)
